src/styles/blocks/card.scss

```diff
@@ -0,0 +1,135 @@
+.card {
+  box-sizing: border-box;
+
+  margin-right: $productDistance;
+  margin-bottom: 46px;
+
+  width: $cardWidth;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__product {
+    display: flex;
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__specs {
+    padding: 0 16px;
+  }
+
+  &__name {
+    color: $mainColor;
+    font-size: 12px;
+    font-weight: $fontWeight-medium;
+    line-height: 150%;
+  }
+
+  &__code {
+    margin: 4px 0 17px;
```

Comment:
let's remind that we should apply either margin-top or -bottom across our entire styles
now some elements have no margin at all, while others have both top and bottom - fix this too